bulk-move.php

```diff
@@ -69,10 +69,39 @@ function load_bulk_move() {
 	$bulk_move->set_plugin_file( __FILE__ );
 
 	add_action( 'plugins_loaded', array( $bulk_move, 'load' ), 101 );
+
```

Comment:
This should be moved inside the `get_ajax_handlers` function. Also the entire AJAX handler should be made into a class. Check `BM_Request_CustomTaxonomyAction` class